cmd/oci-runtime-tool/validate.go

```diff
@@ -27,8 +37,20 @@ var bundleValidateCommand = cli.Command{
 		}
 
 		if err := v.CheckAll(); err != nil {
-			return err
-
+			merr, ok := err.(*multierror.Error)
```

Comment:
It'd be nice if this was a helper function we could reuse in tests.

Reply:
> It'd be nice if this was a helper function we could reuse in tests.

We already have [`FindError`][1], which you can use [like this][2].

I've just pushed 91aa0e9 onto this series adding a `FilterError` helper, in case that's what you're looking for.

[1]: https://github.com/opencontainers/runtime-tools/blob/v0.2.0/specerror/error.go#L148-L152
[2]: https://github.com/opencontainers/runtime-tools/blob/2b007bc3e824086155eeb3cd0f32e522d56369a8/validate/validate_test.go#L170

Reply:
> I've just pushed 91aa0e9…

I've pushed 91aa0e9 → be12cce which flips the order of the `FilterError` response variables to keep `golint` happy (avoiding “error should be the last type when returning multiple items”).